scss/compounds/_category.scss

```diff
@@ -0,0 +1,12 @@
+.nav-link {
```

Comment:
i think i did in the begining to declare by nav-category to add css for padding and color, and i change it to nav link because you said all margin should be same for another nav, i will change to nav-category